packages/eas-cli/src/commands/build/view.ts

```diff
@@ -0,0 +1,76 @@
+import { Command } from '@oclif/command';
+import { HTTPError } from 'got';
+import ora from 'ora';
+
+import { apiClient } from '../../api';
+import { Build } from '../../build/types';
+import formatBuild from '../../build/utils/formatBuild';
+import log from '../../log';
+import {
+  findProjectRootAsync,
+  getProjectAccountNameAsync,
+  getProjectIdAsync,
+} from '../../project/projectUtils';
+
+export default class BuildList extends Command {
+  static description = 'list all builds for your project';
+
+  static args = [{ name: 'buildId' }];
+
+  async run() {
+    const { buildId } = this.parse(BuildList).args;
+
+    const projectDir = (await findProjectRootAsync()) ?? process.cwd();
+    const projectId = await getProjectIdAsync(projectDir);
+    const accountName = await getProjectAccountNameAsync(projectDir);
+
+    const spinner = ora().start('Fetching the build…');
+
+    try {
+      let build: Build;
+
+      if (buildId == null) {
```

Comment:
I think that for this command buildId should required.

Reply:
@wkozyra95 The design doc in notion mentions that we should show last build if no id is specified